Octokit/Models/Response/PullRequest.cs

```diff
@@ -34,7 +34,7 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
             Base = @base;
             User = user;
             MergeCommitSha = mergeCommitSha;
```

Comment:
well. Another quuestion is merge-commit-sha. It is suggested that we use Mergeable instead of merge-commit-sha. Shall we also remove that? 
https://developer.github.com/v3/pulls/#get-a-single-pull-request